OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,18 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.43
+
+## Core
+* Support for azure env HTTPS PROXY provided by addition of variables *azure_proxy_host* and *azure_proxy_port*  in *Opscenterd.conf* (OPSC-17645)
+* Added solr index searcher metrics for solr tables. (OPSC-17680)
+* To compress the backup data, select Enable compression. Compression reduces the amount of data going through your network and reduces the disk and data usage but increases the CPU load for the server. (OPSC-17654)
```

Comment:
We need to clarify that the compression option is being added for Azure. Currently this release not is confusing as the compression option has been there for other destinations for some time. Rather than explain enabling compression and why they would want to, update it to just state that compression can now be enabled on Azure destinations.